llvm/test/CodeGen/LoongArch/lasx/vec-shuffle-bit-shift.ll

```diff
@@ -0,0 +1,200 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Please pre-commit the LASX test cases to better highlight the effects of the optimizations. :)